.github/workflows/nightly_dafny.yml

```diff
@@ -32,3 +32,18 @@ jobs:
     uses: ./.github/workflows/test_models_rust_tests.yml
     with:
       dafny: 'nightly-latest'
+
+  cut-issue-on-failure:
+    runs-on: ubuntu-latest
+    needs: [dafny-nightly-verification, dafny-nightly-java, dafny-nightly-net, dafny-nightly-rust]
+    if: ${{ contains(needs.*.result, 'failure') }}
+    env:
+      GH_TOKEN: ${{ secrets.CI_TOKEN }}
+    steps:
+      - name: Create release blocker on dafny-lang/dafny
+        run: |
+          gh issue create \
```

Comment:
It would be good to create a label for automation-created issues in the Dafny repo, and apply it here with `--label`.

Reply:
That was my original plan, but I found that adding a label to an issue required extra permissions. I figure it's almost as easy for Dafny to search for issues with a prefix rather than a label, and this way the `CI_TOKEN` only needs `public_repo` permissions, which is as minimal/safe as possible.